src/components/Modals/Send/AddressInput/AddressInput.tsx

```diff
@@ -43,11 +50,17 @@ export const AddressInput = ({ rules, placeholder, enableQr = false }: AddressIn
         data-test='send-address-input'
         data-1p-ignore
         // Because the InputRightElement is hover the input, we need to let this space free
-        pe={10}
+        pe={pe}
         isInvalid={!isValid}
+        // This is already a `useCallback()`
+        // eslint-disable-next-line react-memo/require-usememo
+        sx={{
+          fontSize: value && value.length > 42 ? '12px' : '14px',
```

Comment:
i.e `14px` fz is sm, see 
<img width="1358" alt="Screenshot 2024-12-06 at 12 23 06" src="https://github.com/user-attachments/assets/5413369f-d110-4410-9f50-d9bc2b90cd5b">